Could not compile `honggfuzz` Windows 10

Followed all steps
1) https://substrate.dev/docs/en/overview/getting-started/#getting-started-on-windows
2) https://substrate.dev/docs/en/overview/getting-started/#get-the-source

At `cargo build --release`

```
honggfuzz-rs does not currently support Windows but works well under WSL (Windows Subsystem for Linux)
 --> C:\Users\X2\.cargo\registry\src\github.com-1ecc6299db9ec823\honggfuzz-0.5.47\build.rs:7:1
  |
7 | compile_error!("honggfuzz-rs does not currently support Windows but works well under WSL (Windows Subsystem for Linux)");
  | 

```
As on April 2020, Windows has 76.52% market share. Please support Windows

![image](https://user-images.githubusercontent.com/1781048/81387767-dd93cf80-9134-11ea-9d6f-795df7572429.png)

Running `cargo build --release -p node-cli`

gave following error
```

  Compiling sp-debug-derive v2.0.0-dev
error[E0432]: unresolved import `proc_macro`
  --> primitives\debug-derive\src\lib.rs:37:5
   |
37 | use proc_macro::TokenStream;
   |     ^^^^^^^^^^ help: a similar path exists: `syn::proc_macro`

error[E0433]: failed to resolve: use of undeclared type or module `proc_macro`
  --> primitives\debug-derive\src\impls.rs:22:42
   |
22 | pub fn debug_derive(ast: DeriveInput) -> proc_macro::TokenStream {
   |                                          ^^^^^^^^^^ use of undeclared type or module `proc_macro`

error: aborting due to 2 previous errors

Some errors have detailed explanations: E0432, E0433.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `sp-debug-derive`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

At Substrate commit `0947c5b72cebee15e22870cd0087d58727f91aac`
Did
```
rustup default nightly
rustup update nightly
rustup update stable
rustup target add wasm32-unknown-unknown --toolchain nightly
cargo build --release -p node-cli
```
Got the following error
```
error[E0599]: no method named `signal_handler_mut` found for reference `&runtime::Store` in the current scope                                                         
  --> C:\ ..\github.com-1ecc6299db9ec823\substrate-wasmtime-0.16.0-threadsafe.3\src\windows.rs:29:15
   |
29 |         *self.signal_handler_mut() = Some(Box::new(handler));
   |               ^^^^^^^^^^^^^^^^^^ help: there is an associated function with a similar name: `signal_handler`

error: aborting due to previous error                                                                                                                                 

For more information about this error, try `rustc --explain E0599`.
error: could not compile `substrate-wasmtime`.  
```

--- a/client/executor/wasmtime/Cargo.toml
+++ b/client/executor/wasmtime/Cargo.toml
@@ -21,8 +21,8 @@ sp-wasm-interface = { version = "2.0.0-dev", path = "../../../primitives/wasm-in
 sp-runtime-interface = { version = "2.0.0-dev", path = "../../../primitives/runtime-interface" }
 sp-core = { version = "2.0.0-dev", path = "../../../primitives/core" }
 sp-allocator = { version = "2.0.0-dev", path = "../../../primitives/allocator" }
-wasmtime = { package = "substrate-wasmtime", version = "0.16.0-threadsafe.3" }
-wasmtime-runtime = { package = "substrate-wasmtime-runtime", version = "0.16.0-threadsafe.3" }
+wasmtime = { package = "substrate-wasmtime", version = "0.16.0-threadsafe.4" }
+wasmtime-runtime = { package = "substrate-wasmtime-runtime", version = "0.16.0-threadsafe.4" }
 wasmtime-environ = "0.16"
 cranelift-wasm = "0.63"
 cranelift-codegen = "0.63"

